src/vue/common/EmailGetter.vue

```diff
@@ -104,9 +123,60 @@ export default {
         return ''
       }
     },
+
+    async copyToClipboard (data) {
+      try {
+        await navigator.clipboard.writeText(data)
+      } catch (error) {
+        ErrorHandler.processWithoutFeedback(error)
+      }
+    },
   },
 }
 </script>
 
-<style scoped>
+<style lang="scss" scoped>
+  @import "~@scss/variables";
+  @import "~@scss/mixins";
+  .email-getter__wrapper {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .email-getter {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  .copy-button {
```

Comment:
the button should equal to line-height of the email-getter to prevent breaking up of the HTML around. you can use githubs copy button as a reference, you can find it at top of the current page, next to name of the branch.

maybe it worth considering using another ico, since the current one can be hardly visible when it is so small